src/lib/data/collect.ts

```diff
@@ -54,6 +54,11 @@ function processPackageData(rawData: RawPackageData): PackageData {
     return {
       version: rawData.npm.version,
       weeklyDownloads: rawData.downloads.downloads,
+      stableVersioning: {
+        majorVersion: parseInt(rawData.npm.version.split('.')[0], 10) >= 1,
```

Comment:
pull this out to the beginning of the function:

```ts
const [major, minor] = rawData.npm.version.split('.');
```

i know it might not matter because `split` is cheap but practice not calling the same function twice